### DISCUSSION TITLE
动态页面的左侧直播列表

### DISCUSSION THREAD

**Discussion by 596959732** [the1812/Bilibili-Evolved#4017] [提问 / Q&A] [answered]
![image](https://user-images.githubusercontent.com/25106256/220052426-18479d99-1a44-4fdd-b024-4497d2c72c1c.png)
请问这个列表的滚动功能是哪个设置，重装又找不到这个功能了。。。这是我最常用的功能

**Comment by Geeyun-JY3** [comment, marked as answer]
动态首页的正在直播板块相关的功能应该是只有 `直播信息扩充`。

**Reply by 596959732** [reply]
是这个，感谢
